aldryn_forms/admin.py

```diff
@@ -20,10 +21,33 @@
 class FormDataAdmin(admin.ModelAdmin):
 
     date_hierarchy = 'sent_at'
-    list_display = ['__unicode__', 'sent_at']
-    list_filter = ['name', 'people_notified']
+    list_display = ['__unicode__', 'sent_at', 'language']
+    list_filter = ['name', 'language']
     model = FormData
-    readonly_fields = ['name', 'data', 'language', 'sent_at', 'people_notified']
+    readonly_fields = [
+        'name',
+        'data',
+        'language',
+        'sent_at',
+        'get_people_notified'
+    ]
+
+    def has_add_permission(self, request):
+        return False
+
+    def get_people_notified(self, obj):
+        people_list = obj.people_notified.split(':::')
+
+        li_items = [u'<li>{0}</li>'.format(escape(person))
+                    for person in people_list if person]
+
+        if li_items:
+            markup = u'<ul>{0}</ul>'.format(u''.join(li_items))
+        else:
+            markup = ''
+        return markup
+    get_people_notified.allow_tags = True
+    get_people_notified.short_description = _('people notified')
```

Comment:
I don't know.... but without translation context this may become problematic, no?